content/docs/uischema/rules.mdx

```diff
@@ -10,22 +10,22 @@ import ApiLink from '../../../src/components/common/ApiLink';
 
 Rules allow for dynamic aspects for a form, e.g. by hiding or disabling UI schema elements.
 
-A rule may be attached to any UI schema element and can be defined with the `rule` property.
-We'll first look at an example definition of a rule and then explain it in detail.
+A rule may be attached to any UI schema element and can be defined with the `rule` property which looks like:
 
 ```js
 "rule": {
-  "effect": "HIDE",
+  "effect": "HIDE" | "SHOW" | "ENABLE" | "DISABLE",
   "condition": {
-    "scope": "#/properties/name",
-    "schema": { enum: ["foo"] }
+    "scope": "<UI Schema scope>",
+    "schema": JSON Schema
   }
 }
 ```
 
 A rule basically works by first evaluating the `condition` property and in case it evaluates to true, executing the associated `effect`.
 
-## RuleEffect
+
+## Rule Effect
 
 The `effect` property determines what should happen to the attached UI schema element once the `condition` is met. In the example above, if the `name` property has the value of `foo`, we'll hide the UI schema element the rule is attached to. Current effects include:
```

Comment:
This text also needs to be updated ;)

Reply:
Ups missed that, thanks! I just removed the sentence think its clear enough without further description.